fed/proxy/barriers.py

```diff
@@ -188,6 +188,11 @@ async def get_data(self, src_party, upstream_seq_id, curr_seq_id):
         data = await self._proxy_instance.get_data(
             src_party, upstream_seq_id, curr_seq_id
         )
+        if isinstance(data, Exception):
```

Comment:
I think we only need to expose exc_type and src_party, the exception message should be private

Reply:
Indeed, the exception will be wrap to a `RemoteException` that only contains type and src_party and send to other parties